frontend/src/lang/en.ts

```diff
@@ -42,7 +42,7 @@ export const en = {
       gender: {
         female: "Female",
         male: "Male",
-        other: "Non-binary",
+        other: "Other",
```

Comment:
This will probably need to be updated in `es.ts` as well (@Rebecca-LM can help if you need the right word!)

Reply:
https://github.com/CDCgov/prime-simplereport/issues/3163

Reply:
Thanks! I added a note to that ticket to i18nize the `Sex assigned at birth is required` error message as well.